client/src/plugins/axios.js

```diff
@@ -0,0 +1,10 @@
+import globalAxios from 'axios';
+import config from '@/config';
```

Comment:
import { apiUrl } from '@/config'

Reply:
@daniel-pupczyk-rmtcfm-com my config file is something like this `export default { ... }`, so I can't use this syntax. It's looking for named export `apiUrl`. But if I do `module.exports = { ... }` it works.

Reply:
But what's better?

Reply:
Check my comment in config.js